comp/README.md

```diff
@@ -252,6 +252,10 @@ Package client implements a component to send process metadata to the Cluster-Ag
 
 
 
+### [comp/logs/adscheduler](https://pkg.go.dev/github.com/DataDog/datadog-agent/comp/logs/adscheduler)
+
+Package adscheduler contains the AD scheduler component.
```

Comment:
This description doesn't really tell me much that I don't already know from the name. Should we be listing the responsibilities of this component or something more useful here?

Reply:
To be honest, I have limited knowledge about what adscheduer is supposed to do. Any suggestion to improve the description?

Reply:
I also don't really know what `adscheduler` does. Fine to merge as-is, maybe someone who knows can update it in the future

Reply:
AD scheduler is glue code to connect autodiscovery to the logs agent. It receives and filters events and converts them into log sources. 

Relevant logs agent components subscribe to these events for scheduling trailers. 

Ad scheduler filers events and [creates log sources](https://github.com/DataDog/datadog-agent/blob/09c6ece6a0efcb088a63d180de0f2e1e72ad9aac/pkg/logs/schedulers/ad/scheduler.go#L144) -> which get passed to the [source manager](https://github.com/DataDog/datadog-agent/blob/09c6ece6a0efcb088a63d180de0f2e1e72ad9aac/pkg/logs/sources/sources.go#L46) -> Which get passed to [subscriptions](https://github.com/DataDog/datadog-agent/blob/09c6ece6a0efcb088a63d180de0f2e1e72ad9aac/pkg/logs/sources/sources.go#L126) 

- `SubscribeAll` - used by container launcher
- `SubscribeForType` - used by file launcher
- `GetAddedForType` - used by socket launcher

Reply:
Description updated